cli/server_cmd.go

```diff
@@ -66,7 +66,7 @@ var serverCmd = &cobra.Command{
 
 		// setup assets
 		assets, err := mix.New(mix.Config{
-			ManifestFile: "static/mix-manifest.json",
+			ManifestFile: "static/manifest.json",
 			PublicPath:   "/static/",
 		})
```

Comment:
@nics @netsensei Do we still need to keep using the dependency on https://github.com/ugent-library/mix? Kinda weird if we're not using Laravel Mix anymore. In the end it's just loading a JSON file and keeping it as a map for translating assets to their destination path (including md5 hash). Or maybe the manifest file is not even necessary and we just translate the asset path in Go-code and calculate the hash there.

Reply:
no but we need a replacement asset path helper
calculating all the hashes on boot you mean? that seems like unnecessary bookkeeping, when we can just read the manifest?

Reply:
I would say: calculate every hash on first use and cache it from there. Also, I had to manually implement the hash calculation in the manifest myself this time as esbuild does this differently. That's kind of the only added value of the manifest for now I guess.

Reply:
I managed to get the esbuild built-in hashes (in filename) working for te fonts referenced in sass now so calculating md5 hashes ourselves is no longer relevant.